docs/usage.md

```diff
@@ -314,6 +314,8 @@ If `-profile` is not specified, the pipeline will run locally and expect all sof
   - A generic configuration profile to be used with [Charliecloud](https://hpc.github.io/charliecloud/)
 - `apptainer`
   - A generic configuration profile to be used with [Apptainer](https://apptainer.org/)
+- `wave`
+  - A generic configuration profile to enable [Wave](https://seqera.io/wave/) containers. Use together with one of the above (requires Nextflow ` 24.03.0-edge` or later).
```

Comment:
```suggestion
- `seqera_containers`
  - A generic configuration profile to enable [Wave](https://seqera.io/wave/) and to build and pull from the Seqera containers repository. Use together with one of the above (requires Nextflow ` 24.03.0-edge` or later).
```

Reply:
Yep, we can update it in https://github.com/nf-core/rnaseq/pull/1291 first so everything is updated as an example, and then push to the template. Think we have some agreement but will allow time for more discussions.

Reply:
@maxulysse when we have merged this PR can you pull in the changes from `dev` and make this update inb #1291 please?

Reply:
Done https://github.com/nf-core/rnaseq/pull/1291/commits/8b0e4a5575b68912f91fe55143e8a0080538d930